hrs-portlets-webapp/src/main/webapp/WEB-INF/jsp/contactInfo.jsp

```diff
@@ -142,6 +142,12 @@
                      ${prefs['notice'][0]}
                   </p>
                 </c:if>
+                <p class='edit-notice'>
+                  Your preferred name in HRS will have to be updated separately
+                  using the 'Update My Personal Information' link below.
```

Comment:
Rather than telling users to find the link below, we could give them [a clickable hyperlink](https://my.wisc.edu/HRSPortlets/go?urlKey=Personal%20Information) right here.